apps/mobile/src/hooks/balances/use-stx-balances.ts

```diff
@@ -0,0 +1,41 @@
+import { MockedAccount } from '@/mocks/account.mocks';
+import { useStacksSigners } from '@/store/keychains/stacks/stacks-keychains.read';
+
+import { Money } from '@leather.io/models';
+import {
+  useCryptoCurrencyMarketDataMeanAverage,
+  useStxCryptoAssetBalance,
+} from '@leather.io/query';
+import { baseCurrencyAmountInQuote, createMoney, sumMoney } from '@leather.io/utils';
+
+interface StxBalance {
+  stxBalance: Money;
+  stxBalanceUsd: Money;
+}
+
+function useGetAccountStxBalance(account: MockedAccount): Money {
+  const signers = useStacksSigners().fromAccountIndex(account.fingerprint, account.accountIndex);
+  // TODO - ask Kyran if the magic number 0 is correct - signers always returns an array
+  const stxAddress = signers[0]?.address;
```

Comment:
There will only ever be 0 or 1 Stacks addresses per account.

Because this is filtering an array, it returns an array of a single item, but yeah it isn't nice to consume this way. Could this component be wrapped in a loader, such that it accepts the stacks details as a prop?

Reply:
Looking into this and I see now that `useStacksSigners` shares `descriptorKeychainSelectors` with `useBitcoinAccounts` (maybe that should be `useBitcoinSigners`?). 

Rather than refactor that I think it's better to get the address from stacks signers in a cleaner way so I have changed it to this:
```
  const [stacksSigner] = signers;
  const stacksAddress = stacksSigner?.address;
```

That way it can get the first element of the array (in which there is always 0 or 1) without using a magic number